src/sparsezoo/objects/file.py

```diff
@@ -83,7 +83,12 @@ def __init__(
     @property
     def path(self):
         if self._path is None:
-            self.download()
+            cached_path = os.path.join(self.owner_path, self.name)
```

Comment:
we shouldn't use the name "cache" here since the owner/parent doesn't necessarily need to be the cache dir, but functionality should be the same

let's rename to `expected_path` and force a download if the expected path does not exist

Reply:
sounds good.